core/drivers/imx_ocotp.c

```diff
@@ -68,6 +69,21 @@ static inline void ocotp_clock_enable(void) { }
 #error "Platform not supported"
 #endif
 
+#if defined(CFG_MX8M)
```

Comment:
This check needed only for i.MX8M platforms? why not for all platforms ?

Reply:
What platform do you have in mind? i.MX8M (ARMv8-A) are the only ones which support the ARM generic timers. As written in the commit message of:  https://github.com/OP-TEE/optee_os/pull/7453/commits/d2664c6ba21be39214fb89bbb2d945748f5e358a this is an improvement to avoid assumptions. The i.MX9* series is uses another driver since the OCOTP is behind the ELE.

Reply:
I see, my assumption was that archticted-timers are an ARMv8 feature. However, I will include the Cortex-A7 based SoCs as well.